src/components/GraphSection/hooks/useTeamData.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getTeamData, TeamData } from "../../../services/ProjectTeams";
+
+export const useTeamData=( projectId: string)=>
+{
+    const [TeamData, setTeamData] = useState<TeamData[]>([{teamId:projectId , teamName: "Project"}]);
```

Comment:
maintain naming standards
```suggestion
    const [teamData, setTeamData] = useState<TeamData[]>([{teamId:projectId , teamName: "Project"}]);
```